src/deluge/gui/ui/slicer.cpp

```diff
@@ -650,8 +650,14 @@ void Slicer::doSlice() {
 		uint32_t nextDrumStart = lengthInSamples / numClips;
 		firstRange->sampleHolder.endPos = nextDrumStart;
 
-		firstDrum->sources[0].repeatMode =
-		    (lengthMSPerSlice < 2002) ? SampleRepeatMode::ONCE : FlashStorage::defaultSliceMode;
+		if (FlashStorage::defaultShortSliceMode == true) {
```

Comment:
```suggestion
		if (FlashStorage::defaultShortSliceMode) {
```

You never need to compare with a boolean, it's already the right type.

Reply:
right thank you